pages/Documents/RichMessaging/AppleBusinessChatTemplates/custom-interactive-message-template.md

```diff
@@ -0,0 +1,248 @@
+---
+pagename: Custom Interactive Message Template
+Keywords:
+sitesection: Documents
+categoryname: "Rich Messaging"
+documentname: Apple Messages for Business Templates
+permalink: apple-messages-for-business-templates-custom-interactive-message-template.html
+indicator: messaging
+---
+
+### Overview
+
+The Custom Interactive Message (CIM) Template for App Messages for Business allows you to invoke an iOS iMessage app / extension on the consumer device. This app can allow for a wide range of interactivity without requiring the consumer to leave the conversation.
+
+Sending the structured content templates (body and metadata) explained in this document will allow you to send the iMessage app / extension to the consumer device. 
+
+For how to create an iMessage app / extension, see the Apple documentation [here](https://developer.apple.com/imessage/).
+
+### Body Template
+
+The structured content body will allow you to define the layout of the bubble representation in the agent workspace transcript as structured content basic elements.
+
+Please note that the structured content body layout will **not** be represented on the consumer device. The body will help you to create a visual representation in Conversational Cloud for conversational context, transcript and historic records, as well as ease of use for agents.
+
+In order to allow you to report on the number of times an iMessage app was sent from an agent widget during a conversation, make sure to add a unique identifier in the structured content body for each type of iMessage app you use. This will allow you to search transcripts and track how many times a specific app was sent.
+
+For the structured content body fields descriptions and different layout options please refer to [Getting Started with Structured Content.](getting-started-with-rich-messaging-introduction.html)
+
+### Metadata Template
+
+The structured content metadata will allow you to define which iMessage app / extension should be sent to the consumer and which custom data to pass into the iMessage app / extension when it loads.
+
+Using Metadata, you can also control the visual appearance of the bubble that displays to the consumer. This is done with the `receivedMessage` object within the metadata.
+
+#### Metadata Properties
```

Comment:
Optional: We’ve moved to sentence case as our standard for headings. No need to change but FYI.

Reply:
Thanks for the letting me know!